File: Programacao_Para_Dispositivos_Moveis/aula_02/desafio_boasVindas2/app/src/main/java/com/example/boasvindas2/MainActivity.java
package com.example.boasvindas2;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }


    public void boasVindas(View view) {

        EditText nome = findViewById(R.id.nomeTexto);
        EditText saudacao = findViewById(R.id.saudacaoTexto);
        TextView mensagem = findViewById(R.id.mensagem);

        mensagem.setText(saudacao.getText() + ", " + nome.getText() + "\nSeja bem vindo!");


    }
}

File: Programacao_Para_Dispositivos_Moveis/Simulado/app/src/main/java/com/example/avaliacao_parcial/MainActivity.kt
package com.example.avaliacao_parcial

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.EditText
import android.widget.TextView



class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }


    var pares = 0
    var impares = 0
    var numerosDigitados = ""

    fun cadastrarNumeros(view:View) {
        var resultado: TextView = findViewById(R.id.resultado)
        var inserindoNums: EditText = findViewById(R.id.inserindoNums)



        var num: Int = inserindoNums.text.toString().toInt()

        if (num % 2 == 0){
            pares++
            numerosDigitados += "" + num + ", "
        } else {
            impares++
            numerosDigitados += "" + num + ", "
        }

                resultado.text = "O número: " + num + " foi cadastrado!"

    }

    fun mostrarNumeros(view: View){
        var resultado: TextView = findViewById(R.id.resultado)
        resultado.text = "Qtde pares: " + pares + "\nQtde impares: " + impares + "\nNúmeros inseridos: " + numerosDigitados
    }
}